style.css

```diff
@@ -29,7 +29,7 @@ body {
   font-weight: bold;
   font-size: 18px;
```

Comment:
Great job @mesfin345 🤝

- [ ] Kindly make sure the about me section has a background image as this makes your design a replica of the design provided in the Figma template.
- [ ] You may also insert that line just below the background image to make your design similar to the one given in the Figma template.
- Your design
![your contact](https://user-images.githubusercontent.com/63917725/191694740-895ac7c1-b7e3-4d97-bdf1-c5198d6c3ace.PNG)

- Figma design
![Figma contact](https://user-images.githubusercontent.com/63917725/191694821-e69ee125-08d0-43d1-a546-0f6f97a671b6.PNG)